packages/twenty-server/src/engine/metadata-modules/flat-field-metadata/utils/compare-two-flat-field-metadata.util.ts

```diff
@@ -73,15 +82,27 @@ export const compareTwoFlatFieldMetadata = ({
       }
 
       // Remove below assertion when we authorize relation edition, see https://github.com/twentyhq/twenty/commit/39f6f3c4bb101272a9014e142a842d0801a3c33b
-      if (
+      const isRelationFieldType =
         isDefined(fieldMetadata.type) &&
         (fieldMetadata.type === FieldMetadataType.RELATION ||
-          fieldMetadata.type === FieldMetadataType.MORPH_RELATION) &&
-        !['label', 'description', 'isActive'].includes(property)
+          fieldMetadata.type === FieldMetadataType.MORPH_RELATION);
+
+      if (
+        isRelationFieldType &&
+        !relationFlatFieldMetadataPropertiesToCompare.includes(
+          property as RelationFlatFieldMetadataPropertiesToCompare,
+        )
       ) {
         return true;
       }
 
+      const isStandardField =
+        fieldMetadata.standardId !== null && !fieldMetadata.isCustom;
```

Comment:
this is not true to me but there is no consensus in the code base.

1) isCustom <> !isStandard. 
a) A custom is a metadata created by the user. 
b) some metadata are created by the system as the consequence of a custom (for instance base fields on custom, or an FK indexing on a relation). They won't be custom as we don't want the user to edit them but they are not standard either

Reply:
not very important here

Reply:
So we should only be checking standardId definintion ?